ocaml/database/block_device_io.ml

```diff
@@ -515,11 +516,11 @@ let action_writedelta block_dev_fd client datasock target_response_time =
   let generation_count = read_generation_count client in
   read_separator client;
   let length_str = read_length client in
-  let length = int_of_string length_str in
+  let length = int_of_string (Bytes.unsafe_to_string length_str) in
```

Comment:
Since we only read 16 bytes here, would it be okay from a performance point of view to use `to_string`?

Reply:
Yes, but it's overkill. This kind of use of `unsafe_to_string` is pervasive in some ocaml stdlib modules, it's safe to use here and it's not escaping anywhere (also, the danger is really with unsafe_of_string)